accord-core/src/main/java/accord/topology/Topology.java

```diff
@@ -236,7 +236,9 @@ private Topology forSubset(int[] newSubset, Collection<Id> nodes)
         return new Topology(epoch, shards, ranges, nodeLookup, rangeSubset, newSubset);
     }
 
-    private <P1> int[] subsetFor(Unseekables<?, ?> select, IndexedPredicate<P1> predicate, P1 param)
+    private enum OnUnknown { REJECT, IGNORE }
```

Comment:
this was actually a hack to make progress...

What I saw was that the local epoch=1, but we saw epoch=2, we try to see if there is overlap and fail as epoch=1 doesn't have the new table range that is defined in epoch=2

I changed C* code to defer exec of messages and `PreLoadContext` until the epoch they depend on is ready... which may remove the need for this hack...

This likely hasn't been found in burn due to working with a single range, so only would impact new ranges being added and used right away (which happens in C* tests)

Reply:
It kind of makes sense as a work around for the case you are describing, where is this code that tries to compare different epochs that fails?

It seems like this change is opting everything into the `IGNORE` behavior?

Reply:
Went with ignore for all cases as I couldn’t find any caller that allowed the failure. At the dr office so can’t show,  it changing default and running AccordCQLTest you hit this right away

Reply:
here is the exception

```
java.lang.IllegalArgumentException: Range not found for TokenKey{keyspace=demo_ks, key=-9079300900759362424}

	at accord.topology.Topology.subsetFor(Topology.java:266)
	at accord.topology.Topology.visitNodeForKeysOnceOrMore(Topology.java:337)
	at accord.topology.TopologyManager.withUnsyncedEpochs(TopologyManager.java:341)
	at accord.topology.TopologyManager.withUnsyncedEpochs(TopologyManager.java:310)
	at accord.coordinate.CoordinatePreAccept.<init>(CoordinatePreAccept.java:129)
	at accord.coordinate.CoordinateTransaction.<init>(CoordinateTransaction.java:49)
	at accord.coordinate.CoordinateTransaction.coordinate(CoordinateTransaction.java:54)
	at accord.local.Node.initiateCoordination(Node.java:447)
	at accord.local.Node.lambda$coordinate$9(Node.java:439)
	at accord.local.Node.withEpoch(Node.java:220)
	at accord.local.Node.coordinate(Node.java:439)
```

So, the use case is the following

1) create a new Keyspace w/ table
2) run a txn over that new table

The `org.apache.cassandra.distributed.test.accord.NewSchemaTest` test stresses this edge case to make sure the different epochs are supported

Reply:
in this case, `TokenKey` is scoped to the `keyspace`, so a new table in an existing keyspace is fine, but adding a new keyspace and touching it causes this.